docs/user-guide/release-notes/2.17.x.rst

```diff
@@ -14,3 +14,12 @@ New Features
 
 * The export distributor now generates a ``*.iso.DIGESTS`` file along with the ISO.
   This file will contain md5, sha1, sha224, sha256, sha384, and sha512 digests of the ISO.
+
+* RPM content unit dependency solving during e.g recursive copies now utilizes
+  `libsolv` which was added as a `python-pulp-rpm` dependency too.
+
+* The rich/boolean dependency solving is now supported when e.g recursively
+  copying RPM content between repositories.
+
+* More conservative dependency solving is now supported when e.g recursively
```

Comment:
can you explain 'conservative dependency solving'?

Reply:
It's a literal copy from the related issue [headline](https://pulp.plan.io/issues/2478); would  you like me rephrasing it in the release note?